lib/buildWorkspace.nix

```diff
@@ -150,7 +150,7 @@ let
             '';
             buildPhaseCargoCommand = ''
               cargoBuildLog=$(mktemp cargoBuildLogXXXX.json)
-              cargoWithProfile build -p ${crate_name} --message-format json-render-diagnostics >"$cargoBuildLog"
+              cargoWithProfile build ${args.cargoExtraArgs or ""} -p ${crate_name} --message-format json-render-diagnostics >"$cargoBuildLog"
```

Comment:
I don't know why cargoExtraArgs is not called by cargoWithProfile but I'm not very familiar with crane internals. If there's a better fix than this I'm happy to update it

Reply:
cargoWithProfile really is just for setting `--release` or not, so it doesn't check anything else